executable/AndroidManifest.ksy

```diff
@@ -0,0 +1,187 @@
+meta:
+  id: android_manifest
```

Comment:
File name must match id, so it should be "android_manifest.ksy".

Reply:
Parse error: undefined
Call stack: Error
    at $c_Lio_kaitai_struct_format_YAMLParseException.fillInStackTrace__jl_Throwable (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:24379:14)
    at $c_Lio_kaitai_struct_format_YAMLParseException.init___T__jl_Throwable__Z__Z (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:24485:10)
    at $c_Lio_kaitai_struct_format_YAMLParseException.init___T__sci_List (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:40914:58)
    at $c_Lio_kaitai_struct_format_YAMLParseException$.invalidId__T__T__sci_List__Lio_kaitai_struct_format_YAMLParseException (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:29350:63)
    at $c_Lio_kaitai_struct_format_Identifier$.checkIdentifierSource__T__T__sci_List__V (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:7393:61)
    at $c_Lio_kaitai_struct_format_MetaSpec$.fromYaml__O__sci_List__Lio_kaitai_struct_format_MetaSpec (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:28902:11)
    at $c_Lio_kaitai_struct_format_ClassSpec$.fromYaml__O__sci_List__Lio_kaitai_struct_format_MetaSpec__Lio_kaitai_struct_format_ClassSpec (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:28028:83)
    at $c_Lio_kaitai_struct_format_JavaScriptKSYParser$.yamlToSpecs__O__Lio_kaitai_struct_JavaScriptImporter__Lio_kaitai_struct_RuntimeConfig__s_concurrent_Future (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:7636:26)
    at $c_Lio_kaitai_struct_MainJs$.compile__T__sjs_js_Object__Lio_kaitai_struct_JavaScriptImporter__Z__sjs_js_Promise (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:3988:66)
    at $c_Lio_kaitai_struct_MainJs$.$$js$exported$meth$compile__T__sjs_js_Object__Lio_kaitai_struct_JavaScriptImporter__Z__O (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:3942:15)
    at $c_Lio_kaitai_struct_MainJs$.compile (https://ide.kaitai.io/lib/_npm/kaitai-struct-compiler/kaitai-struct-compiler.js:4123:15)
    at CompilerService.compile (https://ide.kaitai.io/js/v1/KaitaiServices.js:85:82)
    at AppController.compile (https://ide.kaitai.io/js/v1/app.js:58:41)
    at AppController.recompile (https://ide.kaitai.io/js/v1/app.js:77:39) io.kaitai.struct.format.YAMLParseException: /meta/id: invalid meta ID: 'android_manifest.ksy', expected /^[a-z][a-z0-9_]*$/